### DISCUSSION TITLE
Getting errors while building on top of TSNScheduleConfigurator

### DISCUSSION THREAD

**Discussion by Abh4git** [inet-framework/inet#941] [General]
I am building libINTET seperately it works fine. But when I extend TSNScheduleConfigurator (inherit from base class and make changes, it throws error while linking using INET 4.5.2 (latest as I see it.)
Creating executable: ../out/clang-release/src/taswiredwifi
ld.lld: error: ../../inet4.5/src/libINET.so: undefined reference to omnetpp::cEvent::setSchedulingPriority(short) [--no-allow-shlib-undefined]
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Any thoughts. I was initially thinking could be due to some old files, but after removing all other files etc, still this is coming. Would be great if someone can provide pointers as I am stuck here. Thank you very much for your time!

**Comment by rhornig** [comment]
Looks like your executable is not linking with the OMNeT++ libraries. Are you sure you have created a proper OMNeT++ project and configured INET project as a project dependency? Proper OMNeT++ project would link with the OMNeT++ libraries autoamtcally.

Try to build the project from the command line and add the `V=1` to the `make` arguments. That will provide more info and you could see the actual linker command line used to link the executable. you should check whether omnet libs are properly added.

**Reply by Abh4git** [reply]
Thank you! It helped. My omnet had some issue. After doing a make or omnetpp things started working.
